### ISSUE TITLE
XML::import($file)->raw() throws an error

### ISSUE THREAD

**Issue by robinsonryan** [ACFBentveld/XML#12]
I just upated to V2.1 from 1.0 and ->raw() throws and error:

`XML::import($file)->raw()`

produces:

`Call to undefined method ACFBentveld\XML\Data\XMLObject::raw()`

I believe the issue is in XML.php in the import() function.

current

```
public static function import(string $path)
    {
        return (new XMLImporter($path))->get();
    }
```

when I change it to this, everything works:

```
public static function import(string $path)
    {
        return (new XMLImporter($path));
    }
``` 
It was a simple enough issue that a PR seemed like overkill.  Great package by the way.

**Comment by AmandoVledder** [comment]
Thanks for pointing out this issue. I've just released v2.1.1 with a fix for this, could you have a look and see if that works for you?

On a side note; 

```php
XML::import($file, true)
```

 now accepts `true` as a second parameter to get the raw xml directly

**Comment by robinsonryan** [comment]
Seems to be working now.
